src/omplexer.ll

```diff
@@ -290,6 +290,9 @@ sizes                     { return SIZES; }
 "}"             { yy_pop_state(); return '}'; }
 ","             { return ','; }
 "\\"            { ; }
+"<="            { return LESSOREQUAL; }
```

Comment:
@XinyaoYI, it seems that these three tokens are never used in the parser. You can double-check. If so, we could just remove them.

If not, could you please insert an underscore between words? It makes reading a bit easier. e.g. `LESS_OR_EQUAL`.